appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -918,183 +923,29 @@ public final void onBrowserEvent(Event event) {
-    pVertPanel.add(projectToolbar);
-    pVertPanel.add(projectListPanel);
-    projectsTabIndex = deckPanel.getWidgetCount();
-    deckPanel.add(pVertPanel);
+    deckPanel.showWidget(0);
+    if ((mayNeedSplash || shouldShowWelcomeDialog())
+                   && !didShowSplash) {
+      showSplashScreens();
+    }
+
+    // Projects tab
+    projectsTabIndex = 0;
 
     // Design tab
-    VerticalPanel dVertPanel = new VerticalPanel();
-    dVertPanel.setWidth("100%");
-    dVertPanel.setHeight("100%");
-
-    // Add the Code Navigation arrow
-//    switchToBlocksButton = new VerticalPanel();
-//    switchToBlocksButton.setVerticalAlignment(VerticalPanel.ALIGN_MIDDLE);
-//    switchToBlocksButton.setHorizontalAlignment(VerticalPanel.ALIGN_CENTER);
-//    switchToBlocksButton.setStyleName("ode-NavArrow");
-//    switchToBlocksButton.add(new Image(RIGHT_ARROW_IMAGE_URL));
-//    switchToBlocksButton.setWidth("25px");
-//    switchToBlocksButton.setHeight("100%");
-
-    // Add the Code Navigation arrow
-//    switchToDesignerButton = new VerticalPanel();
-//    switchToDesignerButton.setVerticalAlignment(VerticalPanel.ALIGN_MIDDLE);
-//    switchToDesignerButton.setHorizontalAlignment(VerticalPanel.ALIGN_CENTER);
-//    switchToDesignerButton.setStyleName("ode-NavArrow");
-//    switchToDesignerButton.add(new Image(LEFT_ARROW_IMAGE_URL));
-//    switchToDesignerButton.setWidth("25px");
-//    switchToDesignerButton.setHeight("100%");
-
-    designToolbar = new DesignToolbar();
-    dVertPanel.add(designToolbar);
-
-    workColumns = new HorizontalPanel();
-    workColumns.setWidth("100%");
-
-    //workColumns.add(switchToDesignerButton);
-
-    Box palletebox = PaletteBox.getPaletteBox();
-    palletebox.setWidth("240px");
-    workColumns.add(palletebox);
-
-    Box viewerbox = ViewerBox.getViewerBox();
-    workColumns.add(viewerbox);
-    workColumns.setCellWidth(viewerbox, "97%");
-    workColumns.setCellHeight(viewerbox, "97%");
-
-    structureAndAssets = new VerticalPanel();
-    structureAndAssets.setVerticalAlignment(VerticalPanel.ALIGN_TOP);
-    // Only one of the SourceStructureBox and the BlockSelectorBox is visible
-    // at any given time, according to whether we are showing the form editor
-    // or the blocks editor. They share the same screen real estate.
-    structureAndAssets.add(SourceStructureBox.getSourceStructureBox());
-    structureAndAssets.add(BlockSelectorBox.getBlockSelectorBox());  // initially not visible
-    structureAndAssets.add(AssetListBox.getAssetListBox());
-    workColumns.add(structureAndAssets);
-
-    Box propertiesbox = PropertiesBox.getPropertiesBox();
-    propertiesbox.setWidth("222px");
-    workColumns.add(propertiesbox);
-    //switchToBlocksButton.setHeight("650px");
-    //workColumns.add(switchToBlocksButton);
-    dVertPanel.add(workColumns);
-    designTabIndex = deckPanel.getWidgetCount();
-    deckPanel.add(dVertPanel);
+    designTabIndex = 1;
```

Comment:
We may not need to fix it now, but technically the design tab is really the project editor tab. When the blocks editor was its own program it really was just design but that's no longer true and a more appropriate name would be helpful.

Reply:
I'm just going to mark as resolved for now since it's not critical for the release.